apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go

```diff
@@ -337,6 +337,9 @@ type ClowdEnvironmentSpec struct {
 	// Defines the default resource requirements in standard k8s format in the
 	// event that they omitted from a PodSpec inside a ClowdApp.
 	ResourceDefaults v1.ResourceRequirements `json:"resourceDefaults"`
+
+	// Node port enables the node port for clowder created services
+	NodePort bool `json:"nodePort,omitempty"`
```

Comment:
Any reason to make this an object and use the `enabled` pattern? I can't think of one off hand, but I'm just crossing t's and dotting i's

Reply:
Based on the [k8s service types ](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types) page, there are other service types besides NodePort and ClusterIP.  So I would be in favor of renaming this field to `serviceType` and making the value a string bound by an enum